perf/src/bin/perf_client.rs

```diff
@@ -308,7 +308,7 @@ async fn request(
 
     let send_stream_stats = stream_stats.new_sender(&send, upload);
 
-    const DATA: [u8; 1024 * 1024] = [42; 1024 * 1024];
+    static DATA: [u8; 1024 * 1024] = [42; 1024 * 1024];
```

Comment:
Why is this? I would usually assume const is better than static.

Reply:
https://rust-lang.github.io/rust-clippy/master/index.html#large_const_arrays